website/docs/guides/athena-qs.md

```diff
@@ -0,0 +1,341 @@
+
+## Set up an Amazon S3 bucket
+
+For the following guide you can use an existing S3 bucket or create a new one.
+
+In either case, download the following CSV files (the Jaffle Shop sample data) and load them into your S3 bucket:
+- [jaffle_shop_customers.csv](https://dbt-tutorial-public.s3-us-west-2.amazonaws.com/jaffle_shop_customers.csv)
+- [jaffle_shop_orders.csv](https://dbt-tutorial-public.s3-us-west-2.amazonaws.com/jaffle_shop_orders.csv)
+- [stripe_payments.csv](https://dbt-tutorial-public.s3-us-west-2.amazonaws.com/stripe_payments.csv)
+
+
+## Configure Amazon Athena
+
+1. Log into your AWS account and navigate to the **Athena console**.
+    - If this is your first time in the Athena console (in your current AWS Region), click **Explore the query editor** to open the query editor. Otherwise, Athena opens automatically in the query editor.
+1. Open **Settings** and find the **Location of query result box** field.
+    1. Enter the path of the S3 bucket (prefix it with `s3://`).
+    2. Navigate to **Browse S3**, select the S3 bucket you created, and click **Choose**.
+1. **Save** the settings configurations
+1. In the **query editor**, create a database by running `create database YOUR_DATABASE_NAME`.
+1. To make the database you created the one you `write` into, select it from the **Database** list on the left side menu. 
+1. There are 2 options to access the Jaffle Shop data in the S3 bucket:
+    1. Manually create the tables
+    2. Create a glue crawler to recreate the data as external tables (recommended)
+1. Once the tables have been created, you will able to `SELECT` from them. 
+
+## Set up security access to Athena
+
+To setup the security access for Athena, determine which access method you want to use: 
+    1. The `aws_access_key_id` and `aws_secret_access_key` (recommended)
+    2. An `aws_profile_name` that matches your profile defined in your **AWS credentials** file.
+
+### AWS access key
+
+To obtain your `aws_access_key_id` and `aws_secret_access_key`:
+
+1. Open the **AWS Console**.
+1. Click on your **username** near the top right and click **Security Credentials**.
+1. Click on **Users** in the sidebar.
+1. Click on your **username** (or the name of the user for whom to create the key).
+1. Click on the **Security Credentials** tab.
+1. Click **Create Access Key**.
+1. Click **Show User Security Credentials** and 
+
+Save the `aws_access_key_id` and `aws_secret_access_key` for a future step.
+
+### AWS profile name
+
+To obtain your AWS credentials file:
+1. Follow the instructions for [configuring the credentials file](https://docs.aws.amazon.com/cli/v1/userguide/cli-configure-files.html) usin the AWS CLI
+1. Locate the file on your computer
+    1. Windows: `%USERPROFILE%\.aws\credentials`
+    2. Mac/Linux: `~/.aws/credentials`
+
+Save the `aws_access_key_id` and `aws_secret_access_key` from the file for a future step.
```

Comment:
```suggestion
1. Save the `aws_profile_name` and password from the downloaded file for a future step.
```

Reply:
The original was correct. Those fields are what they retrieve from the file

Reply:
These are the fields that would be in the file. I messed up and neglected to name the file above.